cves/kernel/CVE-2021-43267.yml

```diff
@@ -147,10 +150,12 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    This vulnerability was discovered by SentinelOne researcher named Max van
```

Comment:
May possibly be a good idea to avoid directly naming individuals.

Reply:
fixed